services/web/server/src/simcore_service_webserver/projects/db.py

```diff
@@ -730,6 +744,7 @@ async def get_project(
         projects.c.published,
         projects.c.hidden,
         projects.c.workspace_id,
+        projects.c.trashed_at,
     ]
 
     async def get_project_db(self, project_uuid: ProjectID) -> ProjectDB:
```

Comment:
`get_project_db` Here we return the pure project DB `ProjectDB` and it is using the `_SELECTION_PROJECT_DB_ARGS` so I would add it to the model.